### DISCUSSION TITLE
Ability to Add/Edit Poster and Banner Images

### DISCUSSION THREAD

**Discussion by gmusgrave** [pymedusa/Medusa#7551] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
Some shows don't have poster and/or banner images. Doing a "_Force Full Update_" doesn't seem to grab them, even days after the show was added.

Very rarely, the poster/banner will be incorrect (i.e.: for a different show).

Almost invariably, I can find appropriate artwork on the internet, and it would be nice to add it to Medusa.

Examples of recently added shows (in the last week) that have no art: 

1. The Masked Singer (UK) - ITV
2. Deadwater Fell - Channel 4
3. Cobra (2020) - Sky1
4. October Falcon - Netflix

**Describe the solution you'd like**
A means of adding or editing the poster and banner images for a given show. Could be in the "_Edit | Advanced_" panel, or available by clicking on the image.

In either case, a dialog for uploading the image would be available.

**Comment by Maydeedvd** [comment]
Hi,
Same issues with "You" and "The outsider"
At least, allow to change the indexer.
Thx

**Comment by p0psicles** [comment]
You can change the indexer. Remove show, but keep files. Add show

**Comment by Maydeedvd** [comment]
I know
But, Agreed it's not a good way to do. ^^
Thx
